api/models.py

```diff
@@ -817,32 +817,8 @@ def get_sender_addr(self):
     def get_sender_name(self):
         return self.sender[0]
 
-    def _get_age(self, date):
-        def _seconds_to_human(sec):
-            unit = "second"
-            if sec > 60:
-                sec /= 60
-                unit = "minute"
-                if sec > 60:
-                    sec /= 60
-                    unit = "hour"
-                    if sec > 24:
-                        sec /= 24
-                        unit = "day"
-                        if sec > 7:
-                            sec /= 7
-                            unit = "week"
-            if sec >= 2:
-                unit += "s"
-            return "%s %s" % (int(sec), unit)
-
-        age = int((datetime.datetime.utcnow() - date).total_seconds())
-        if age < 0:
-            return "now"
-        return _seconds_to_human(age)
-
     def get_age(self):
-        return self._get_age(self.date)
+        return self.date
```

Comment:
The method now no longer returns an "age", it's raw date. We can drop this method and let callers access `.date`.